rosetta-source/src/main/rosetta/event-workflow-type.rosetta

```diff
@@ -41,6 +41,7 @@ type WorkflowStep: <"A workflow step represents the state of a business event. T
     [rootType]
 
     businessEvent BusinessEvent (0..1) <"Life cycle event for the step. The businessEvent is optional when a proposedEvent or rejection are present.">
+    counterpartyPositionBusinessEvent CounterpartyPositionBusinessEvent (0..1) <"Documents the life cycle event for a position.">
```

Comment:
@rogerguitart new property counterpartyPositionBusinessEvent is bring added here under counterpartyPositionBusinessEvent. my gues is its because of this its being added to samples

Reply:
Okay, but there is also the businessEvent inside WorkflowStep, but that one doesn't appear in every sample under these folders, since both businessEvent and counterpartyPositionBusinessEvent are optional (for example, see the sample `msg ex19 cds execution allocations` under the fpml-5-10/incomplete-processes folder). Maybe I am missing technical knowledge...